README.md

```diff
@@ -30,112 +30,100 @@ To facilitate robust API governance, we're excited to introduce **API Analysis**
-- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
+- [Azure Functions Core Tools v4.0.5382 or above](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Cisolated-process%2Cnode-v4%2Cpython-v2%2Chttp-trigger%2Ccontainer-apps&pivots=programming-language-typescript#install-the-azure-functions-core-tools).
+- [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd).
+- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli).
 
-### :cloud: How to deploy
-
-Follow the instructions in [Quickstart: Create a function in Azure with TypeScript using Visual Studio Code](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-typescript?pivots=nodejs-model-v4#sign-in-to-azure). Start from the "Sign in to Azure" section and complete all subsequent sections.
+### :wrench: Running the sample using the Azure Developer CLI (azd)
 
-Alternatively, if you'd like to use [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd), run the following steps:
+The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
 
-1. Login to Azure by running the commands below in your terminal:
+After logging in with the following command, you will be able to use the `azd` cli to quickly provision and deploy the application.
 
-    ```bash
-    az login
-    azd auth login
-    ```
+```
+# authenticate to Azure Developer CLI
+azd auth login
 
-1. Initialize your Azure environment by running the command below:
+# authenticate to Azure CLI
+az login
+```
 
-    ```bash
-    azd init
-    ```
+Initialize your Azure environment.
+```
+azd init
+```
+Enter an environment name.
 
-   While you're running `azd init`, you will be asked to provide:
+Then, run `azd up` to provision all the resources to Azure and deploy the code to those resources.
+```
+azd up
+```
 
-   - How to initialize: Choose the `Select a template` option, then `Y`.
-   - A template: Choose the `Minimal template` option. <!-- This option will be changed to "API Center Analyzer" once this PR is merged, which is not available at this time of PR -->
-   - An environment name: Any name you want to give to your environment.
+Select your desired `subscription` and `location`. Then choose whether to create the monitoring resources. If you want to use an existing api center resource, pass the values for `apiCenterName` and `apiCenterResourceGroupName`. Otherwise leave them blank to create a new one. Wait a moment for the resource deployment to complete. Then you can upload your own api definition for test.
 
-1. Run the following command to provision and deploy this analyzer to Azure:
+(Optional) You can also set the Env variables manually using the following command to skip setting them during the `azd up` process.
+```
+azd env set USE_MONITORING <yes/no>
```

Comment:
this should be true/false, not yes/no

Reply:
Well, I respectfully against that. `yes/no` might be more intuitive than `true/false`. That value can easily be convertible to `true/false` within bicep.

Reply:
It's your template and you can decide this, but we do have a precedent for bool values that enable features to true/false here: https://github.com/Azure-Samples/todo-nodejs-mongo/blob/main/infra/main.parameters.json#L15

and in other templates.

I'd prefer to be consistent across templates.  But your call.